docs/getting-started/upgrade/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Upgrade Shopify App CLI
+section: getting-started
+toc: false
+redirect_from: "/upgrade/"
+---
+
+You can manage upgrades to Shopify App CLI with the package manager for your platform.
+
+### Brew (MacOS)
+
+```console
+$ brew update && brew upgrade shopify-cli
+```
+
+### apt (Debian, Ubuntu)
+
+```console
+$ sudo apt update
+$ sudo apt upgrade shopify-cli
+```
+
+### RPM (CentOS, Fedora, Red Hat, SUSE)
```

Comment:
To align with heading in install page - "RPM-based Linux distributions (e.g., Fedora, CentOS, SUSE)"